common/attest_plugin.h

```diff
@@ -20,6 +20,21 @@
 
 OE_EXTERNC_BEGIN
 
+/**
+ * Note: V1 is OE_REPORT_HEADER_VERSION, for legacy report header
+ * of type oe_report_header_t.
+ *
+ * V2 is for legacy attestation header of type oe_attestation_header_t.
+ * For SGX local and remote attestation, the evidence requires a legacy
+ * report header of type oe_report_header_t to prefix the SGX report or
+ * quote.
+ *
+ * V3 is the current version. Its also for attestation header of type
+ * oe_attestation_header_t. SGX report or quote will not be prefixed with
+ * a legacy header of type oe_report_header_t.
+ */
+#define OE_ATTESTATION_HEADER_VERSION (3)
```

Comment:
I agree it's a good idea to bump this and the documentation of the versioning changes is useful. I would additionally suggest adding a statement that only the latest version is supported, so that it's clear that the lack of downlevel checks in `oe_verify_evidence` is intentional.

Reply:
Thanks! Added the suggested comment.